tested/evaluators/exception.py

```diff
@@ -18,9 +18,11 @@ def try_as_exception(value: str) -> Either[ExceptionValue]:
         return Either(e)
 
 
-def try_as_readable_exception(value: str) -> Optional[str]:
+def try_as_readable_exception(_config: EvaluatorConfig,
+                              value: str) -> Optional[str]:
     try:
         actual = ExceptionValue.parse_raw(value)
+        actual = _config.bundle.lang_config.exception_output(actual)
```

Comment:
The underscore before the variable name is conventionally used when the variable is not used, but we do use it here, so it's better to remove the underscore.